rust/README.md

```diff
@@ -32,21 +29,25 @@ path = bridgestan.compile_model("stan_model.stan", make_args=["STAN_THREADS=true
 If `STAN_THREADS` was not specified while building the model, the Rust wrapper
 will throw an error when loading the model.
 
-## Usage:
+## Usage
 
-Run this example with `cargo run --example=example`.
+Run this example with `RUST_LOG=info cargo run --example=example --features download-bridgestan-src`.
 
 ```rust
 use std::ffi::CString;
 use std::path::Path;
-use bridgestan::{BridgeStanError, Model, open_library};
+use bridgestan::{BridgeStanError, Model, open_library, compile_model, download_bridgestan_src};
 
-// The path to the compiled model.
-// Get for instance from python `bridgestan.compile_model`
+// The path to the Stan model
 let path = Path::new(env!["CARGO_MANIFEST_DIR"])
     .parent()
     .unwrap()
-    .join("test_models/simple/simple_model.so");
+    .join("test_models/simple/simple.stan");
+
+let bs_path = download_bridgestan_src().unwrap();
+// The path to the compiled model
+let path = compile_model(bs_path, path, vec![], vec![]).expect("Could not compile Stan model.");
```

Comment:
Could we do something similar to the other examples and have this be a relative path (with the download option commented out as an example?)?

E.g.:
https://github.com/roualdes/bridgestan/blob/ef1f98694b9c6236e2d0b96eba025e864150e2de/julia/example.jl#L5-L9